block/manager_test.go

```diff
@@ -79,7 +79,7 @@ func TestInitialState(t *testing.T) {
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
 			assert := assert.New(t)
-			logger := test.NewFileLoggerCustom(t, test.TempLogFileName(t, c.name))
+			logger := test.NewFileLoggerCustom(t, test.TempLogFileName(t))
```

Comment:
If you are removing this custom name you should be able to just call NewFileLogger.